quartical/executor.py

```diff
@@ -89,6 +89,8 @@ def _execute(exitstack):
         # less convenient pattern.
         client.run_on_scheduler(install_plugin)
 
+        client.wait_for_workers(dask_opts.workers)
```

Comment:
Looks good to me. I think we're going to have to pass workers on the command line. Does doing so override user_config.yaml? I suspect it does and this is desired behaviour, just checking.

Reply:
Should do - priority is always defaults < .yaml < cli, so anything specified on command line will take precedence.